### PR TITLE
Align result to SSE when input is 0.0f/-0.0f in _mm_rsqrt_{ps, ss}

### PR DESCRIPTION
**Pull request by Cuda-Chen**
Align result to SSE when input is 0.0f/-0.0f in `_mm_rsqrt_{ps, ss}`.

### CODE REVIEW

**Comment by jserv** [comment]
Can you please provide some relevant information in `README.md`? Therefore, developers can be aware of such floating-point compatibility.

**Comment by Cuda-Chen** [comment]
> Can you please provide some relevant information in `README.md`? Therefore, developers can be aware of such floating-point compatibility.

Yes. Let me come up with some sentences for explaining this.

File: README.md [hunk 1]
```diff
@@ -36,6 +36,31 @@ their equivalents are built utilizing a number of NEON intrinsics.
 For example, SSE intrinsic `_mm_loadu_si128` has a direct NEON mapping (`vld1q_s32`),
 but SSE intrinsic `_mm_maddubs_epi16` has to be implemented with 13+ NEON instructions.
 
+### Floating-point compatibility
+
+Some conversions require serveral NEON intrinsics, which may produce inconsistent
+results compared to their SSE counterparts because of arthimetic rules of IEEE-754.
```
Comment (jserv) [inline README.md:42]: Refine as following:
> Some conversions require several NEON intrinsics, which may produce inconsistent results compared to their SSE counterparts due to differences in the arithmetic rules of IEEE-754.

File: README.md [hunk 2]
```diff
@@ -36,6 +36,31 @@ their equivalents are built utilizing a number of NEON intrinsics.
...
+    float32x4_t out = vrsqrteq_f32(vreinterpretq_f32_m128(in));
+
+    out = vmulq_f32(
+        out, vrsqrtsq_f32(vmulq_f32(vreinterpretq_f32_m128(in), out), out));
+
+    return vreinterpretq_m128_f32(out);
+}
+```
+
+The `_mm_rsqrt_ps` conversion will produce NaN if a source value is 0.0 (first Inf
```
Comment (jserv) [inline README.md:58]: Proofread as following:
> The `_mm_rsqrt_ps` conversion will produce NaN if a source value is `0.0` (first INF for the reciprocal square root of 0.0, then INF * `0.0` using `vmulq_f32`). In contrast, the SSE counterpart produces INF if a source value is `0.0`. As a result, additional treatments should be applied to ensure consistency between the conversion and its SSE counterpart.

Comment (Cuda-Chen) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (Cuda-Chen) [reply README.md]: Done.

**Comment by jserv** [comment]
Thank @Cuda-Chen for contributing!

**Pull request merged**